src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const onlyMan = century
+    ? people.filter(person => (
+      person.sex === 'm' && century === Math.ceil(person.died / 100)))
+    : people.filter(person => person.sex === 'm');
+
+  const onlyManAges = onlyMan.map(person => person.died - person.born);
+
+  return onlyManAges.reduce((a, b) => a + b) / onlyManAges.length;
```

Comment:
Map is redundant here, you can calculate (person.died - person.born) in reduce method